pkgs/by-name/wg/wg-access-server/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildGoModule
+, buildNpmPackage
+, fetchFromGitHub
+, makeWrapper
+, iproute2
+}:
+
+buildGoModule rec {
+  pname = "wg-access-server";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "freifunkMUC";
+    repo = "wg-access-server";
+    rev = "v${version}";
+    hash = "sha256-m7Zf8LAkVgFHBNgCr8TZTtp28LH2tFRLhdpulp//pbM=";
+  };
+
+  proxyVendor = true; # darwin/linux hash mismatch
+  vendorHash = "sha256-aEJu3dTKyu2d/xI03DEeOqNl0nkAPkURF/AK7mOf5A4=";
+
+  CGO_ENABLED = 1;
+
+  ldflags = [ "-s" "-w" ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  checkFlags = [ "-skip=TestDNSProxy_ServeDNS" ];
+
+  ui = buildNpmPackage {
+    inherit version src;
+    pname = "wg-access-server-ui";
+
+    npmDepsHash = "sha256-NDjADRnJHOA2gbj7Ah5BQkFZPbqx2vZjarQeO4LBITI=";
+
+    sourceRoot = "${src.name}/website";
+
+    installPhase = ''
+      mkdir -p $out
+      mv build/ $out/site
```

Comment:
As far as I can tell, [upstream code](https://github.com/freifunkMUC/wg-access-server/blob/4e6007e66a904eba374879b0814ef313c1ac0741/internal/services/website_router.go#L20) looks for `website/build` relative to the current working directory.

Does this currently work? I think we need to
- Change the ui output to look like `.../website/path`
- Add a patch to modify the line I linked above to hardcode the ui output path
- Remove the part installing the site from postInstall

Reply:
I've handled this in a yet to be upstreamed nixos module for wg-access-server. But i agree that the ui should be copied into the output.